sentry-spring/src/main/java/io/sentry/spring/SentryUserProviderEventProcessor.java

```diff
@@ -13,6 +13,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+// TODO: check if I need to support process(SentryTransaction) here
```

Comment:
@maciejwalkowiak

Reply:
does it make sense to run this event processor for transactions too?

Reply:
If it would run for transactions we would not need to add user filter - so I guess yes